src/services/__tests__/tourettes.test.js

```diff
@@ -18,31 +18,49 @@ describe('tourettes', () => {
   describe('sayTourettes', () => {
     describe('when input text is empty', () => {
       const inputText = '';
+      const minAmountOfReplies = 3;
 
       it('replies with at least three items from the profanities list', () => {
         const reply = sayTourettes(inputText);
+        const amountOfReplies = reply.split(' ').length;
         expect(reply).toNotBe(undefined);
-        expect(reply.split(' ').length).toBeGreaterThan(3);
+        expect(amountOfReplies).toBeGreaterThanOrEqualTo(minAmountOfReplies);
+      });
+    });
+
+    describe('when input text is integer', () => {
+      const amountOfProfanities = 25;
```

Comment:
I agree, preferably not a known number anyway for the same reason. Maybe a random from 3 to 10 here as well?
I'd like to avoid 1 since it's the default reply after one word.